src/main/java/org/sunbird/common/util/EsClientFactory.java

```diff
@@ -0,0 +1,33 @@
+package org.sunbird.common.util;
+
+import org.sunbird.common.service.ElasticSearchService;
+import org.sunbird.common.service.impl.ElasticSearchRestHighImpl;
+
+public class EsClientFactory {
```

Comment:
This is not required. ElasticSearchRestHighImpl should be annotated as service or component and autowired by other classes